src/test/java/org/skife/jdbi/v2/sqlobject/TestGetGeneratedKeysPostgres.java

```diff
@@ -67,9 +72,10 @@ public Object withHandle(Handle handle) throws Exception
         public String findNameById(@Bind long id);
     }
 
-    @Ignore
     @Test
     public void testFoo() throws Exception {
+        assumeTrue(postgresIsInstalled);
```

Comment:
Maybe put the assumption in a `@BeforeClass`? Then neither your `@Before` or `@After` methods will ever be called.